bodhi/client/__init__.py

```diff
@@ -16,12 +16,37 @@
 import platform
 import subprocess
 import sys
-import traceback
 
 import click
 
 from bodhi.client import bindings
 
+common_options = [
+    click.option('--user', envvar='USERNAME'),
+    click.option('--password', prompt=True, hide_input=True),
+    click.option('--type', default='bugfix', help='Update type', required=True,
+                 type=click.Choice(['security', 'bugfix', 'enhancement', 'newpackage'])),
+    click.option('--notes', help='Update description'),
+    click.option('--bugs', help='Comma-seperated list of bug numbers', default=''),
+    click.option('--close-bugs', default=True, is_flag=True, help='Automatically close bugs'),
+    click.option('--request', help='Requested repository',
+                 type=click.Choice(['testing', 'stable', 'unpush'])),
+    click.option('--autokarma', default=True, is_flag=True, help='Enable karma automatism'),
+    click.option('--stable-karma', type=click.INT, help='Stable karma threshold'),
+    click.option('--unstable-karma', type=click.INT, help='Unstable karma threshold'),
+    click.option('--suggest', help='Post-update user suggestion',
+                 type=click.Choice(['logout', 'reboot'])),
+    click.option('--staging', help='Use the staging bodhi instance',
+                 is_flag=True, default=False)]
+
+
+def add_options(options):
```

Comment:
Let's give this wrapper a docblock that explains its purpose, options argument, and return value.